fill/BUILD

```diff
@@ -2,7 +2,9 @@ cc_library(
     name = "telemetry_reader",
     srcs = ["telemetry_reader.cc"],
     hdrs = ["telemetry_reader.h"],
-    deps = ["//protos:telemetry_cc_proto"],
+    deps = [
+        "//protos:telemetry_cc_proto",
```

Comment:
At some point, we should probably distinguish (in name) between fill station telemetry and rocket telemetry